docs/source/using-discovery/workingwithgpu.rst

```diff
@@ -86,22 +114,33 @@ limit on each job. Both partitions give you access to all of the above GPU types
      - 12
      - 12
```

Comment:
Do we need to have the Core Limit and RAM Limit listed in this table since they are both N/A. It would shrink the table up a little bit. Also would there be confusion around that you can have 25 running jobs but only can use 8 GPUs on the gpu partition and only 12 gpus on the multigpu partition?

Reply:
@joseph-cooke , I checked with @gshomo and have removed the "Running Jobs" column from the table